packages/desktop-client/src/components/settings/Encryption.js

```diff
@@ -19,30 +22,41 @@ export default function EncryptionSettings({ prefs, pushModal }) {
           End-to-end Encryption is turned on.
         </Text>{' '}
         Your data is encrypted with a key that only you have before sending it
-        out to the cloud . Local data remains unencrypted so if you forget your
-        password you can re-encrypt it.
+        it out to the cloud. Local data remains unencrypted so if you forget
+        your password you can re-encrypt it.{' '}
+        <a href="https://actualbudget.github.io/docs/Getting-Started/sync/#encryption">
+          Learn more…
+        </a>
       </Text>
     </ButtonSetting>
-  ) : (
+  ) : serverURL ? (
     <ButtonSetting
       button={
-        <Button
-          onClick={() => {
-            alert(
-              'End-to-end encryption is not supported on the self-hosted service yet'
-            );
-            // pushModal('create-encryption-key');
-          }}
-        >
+        <Button onClick={() => pushModal('create-encryption-key')}>
           Enable encryption…
         </Button>
       }
     >
       <Text>
-        <strong>End-to-end encryption</strong> is not enabled. Any data on our
-        servers is still stored safely and securely, but it's not end-to-end
-        encrypted which means we have the ability to read it (but we won't). If
-        you want, you can use a password to encrypt your data on our servers.
+        <strong>End-to-end encryption</strong> is not enabled. Any data on the
+        server is still protected by the server password, but it's not
+        end-to-end encrypted which means the server owners have the ability to
+        read it. If you want, you can use an additional password to encrypt your
+        data on the server.{' '}
+        <a href="https://actualbudget.github.io/docs/Getting-Started/sync/#encryption">
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
        <a href="https://actualbudget.github.io/docs/Getting-Started/sync/#encryption" target="_blank">
```